src/api/api.js

```diff
@@ -1 +1,33 @@
-export const BASE_URL = 'https://mate-api.herokuapp.com';
+/* eslint-disable */
+const BASE_URL = 'https://mate-api.herokuapp.com';
+
+export const request = url => fetch(`${BASE_URL}${url}`)
+  .then(response => response.json())
+  .then(result => result.data);
+
+export const getPosts = () => request('/posts');
+export const getUsers = () => request('/users');
+
+export const getComments = postId => request('/comments')
+  .then(response => response.filter(comment => (
+    comment.postId === postId
+  )));
```

Comment:
It would be good to move it to separate file comments.js (as task description says:
>Create getPostComments(postId) method in /src/api/comments.js, load comments from /comments and filter them using a given postId